addon/components/bourbon-search-select-field.js

```diff
@@ -163,15 +166,14 @@ export default Component.extend(SelectMixin, {
           this.set('searchList', A(['No results found.']));
         }
       } else {
+        this.set('noResults', false)
```

Comment:
You're only setting `noResults` in the `else` block here, so if it hits the first part then this value won't change even if content is valid right? Or i'm just missing something. Honestly you can also revert this if you want because I get what you were trying to do now

Reply:
I am setting `noResults` to true on line 160.